### PR TITLE
ARM OSX Migrator

### PR DESCRIPTION
**Pull request by tschuelia**
continue PR https://github.com/conda-forge/r-maps-feedstock/pull/14

### CODE REVIEW

**Comment by tschuelia** [comment]
@conda-forge-admin please rerender

**Comment by tschuelia** [comment]
As stated in PR https://github.com/conda-forge/r-maps-feedstock/pull/14 the issue seems to be related to the CPU type with R
If I understand everything correctly, the CI runs on an x86 machine and tries to cross-compile to OSX ARM. The maps package tries to build some lib called `shlib` using an `R_HOME` environment variable (see [here](https://github.com/adeckmyn/maps/blob/dc1eefb14d155fca33ace2333c490f5fd52eeb34/src/Makefile.in#L73)) and this `R_HOME` seems to try to use some OSX ARM R version that of course does not run on x86

edit: building locally on my M1 Mac works perfectly fine, so I'm hoping that this is the only issue

**Review by ocefpaf — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -16,15 +15,15 @@ source:
 build:
   merge_build_host: true  # [win]
   number: 0
```
Comment (dbast) [inline recipe/meta.yaml]: Please bump the version number, as e.g. the license spdx update is for all platforms
```suggestion
  number: 1
```

**Comment by dbast** [comment]
closes https://github.com/conda-forge/r-maps-feedstock/pull/14

**Pull request merged**